.travis/before_install.sh

```diff
@@ -1,81 +0,0 @@
-#!/usr/bin/env bash
-
-# WARNING: DO NOT EDIT!
-#
-# This file was generated by plugin_template, and is managed by bootstrap.py. Please use
-# bootstrap.py to update this file.
-#
-# For more info visit https://github.com/pulp/plugin_template
-
-set -mveuo pipefail
-
-export PRE_BEFORE_INSTALL=$TRAVIS_BUILD_DIR/.travis/pre_before_install.sh
-export POST_BEFORE_INSTALL=$TRAVIS_BUILD_DIR/.travis/post_before_install.sh
-
-COMMIT_MSG=$(git log --format=%B --no-merges -1)
-export COMMIT_MSG
-
-if [ -f $PRE_BEFORE_INSTALL ]; then
-    $PRE_BEFORE_INSTALL
-fi
-
-if [[ -n $(echo -e $COMMIT_MSG | grep -P "Required PR:.*" | grep -v "https") ]]; then
```

Comment:
Is all of the Required PR functionality done via github actions?

Reply:
it was included just to have commit validation,
I'm working on a replacement for TRAVIS_COMMIT_RANGE, and probably we won't need this file

Reply:
To clarify, it was copied over from other repos because the commit validation code was mixed in with it, but it doesn't really serve a purpose here.

https://github.com/pulp/ansible-pulp/commit/284ab970250e261ea1d2d14e08246249068a1472#diff-772839f7e87e4ac33eb4bed22ec2f68f